docs/logs.mdx

```diff
@@ -0,0 +1,119 @@
+        ✅   5. isPermissionDialogVisible (native)
+        ✅   6. tap widgets with text "Request camera permission".
+        ✅   7. isPermissionDialogVisible (native)
+        ⏳   8. denyPermission (native)
+❌ denies various permissions (integration_test/permissions/deny_many_permissions_twice_test.dart) (9s)
+══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════
+The following PlatformException was thrown running a test:
+PlatformException(PermissionHandler.PermissionManager, A request
+for permissions is already running, please wait for it to finish
+before doing another request (note that you can request multiple
+permissions at the same time)., null, null)
+
+When the exception was thrown, this was the stack:
+#0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:648:7)
+#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:334:18)
+<asynchronous suspension>
+#2      MethodChannelPermissionHandler.requestPermissions (package:permission_handler_platform_interface/src/method_channel/method_channel_permission_handler.dart:79:9)
+<asynchronous suspension>
+#3      PermissionActions.request (package:permission_handler/permission_handler.dart:52:31)
+<asynchronous suspension>
+#4      _PermissionsScreenState._requestCameraPermission (package:e2e_app/permissions_screen.dart:21:20)
+<asynchronous suspension>
+
+The test description was:
+  denies various permissions
+═════════════════════════════════════════════════════════════════
+
+✅ taps on notification (integration_test/permissions/notifications_test.dart) (16s)
+✅ taps on notification native2 (integration_test/permissions/notifications_test.dart) (14s)
+✅ grants various permissions (integration_test/permissions/permissions_many_test.dart) (15s)
+...
+```
+
+After tests have ended, there is a summary printed:
+
+```
+Test summary:
+📝 Total: 8
+✅ Successful: 3
+❌ Failed: 5
+  - taps on notification (integration_test/permissions/notifications_test.dart)
+  - taps on notification native2 (integration_test/permissions/notifications_test.dart)
+  - accepts location permission (integration_test/permissions/permissions_location_test.dart)
+  - accepts location permission native2 (integration_test/permissions/permissions_location_test.dart)
+  - grants various permissions (integration_test/permissions/permissions_many_test.dart)
+⏩ Skipped: 0
+📊 Report: file:///Users/user/patrol/dev/e2e_app/build/app/reports/androidTests/connected/index.html
+⏱️  Duration: 227s
+```
+
+## Disabling logs
+
+You can use following flags to configure, which logs should be printed. Pass them to `patrol test` or `patrol develop` command.
+ - --[no-]show-flutter-logs -> Show Flutter logs while running the tests. (Available only in `patrol test`)
+ - --[no-]hide-test-steps -> Hide test steps while running the tests.
+ - --[no-]clear-test-steps -> Clear test steps after the test finishes.
```

Comment:
It will be useful to add here default values (for clear-test-steps it's true).
Maybe this should be a table with columns:
- flag
- default value
- available in test/develop
- description